pkg/reconciler/v1alpha1/pipelinerun/pipelinerun_test.go

```diff
@@ -409,3 +431,16 @@ func TestReconcileOnCompletedPipelineRun(t *testing.T) {
 		t.Errorf("Expected PipelineRun status to be complete, but was %v", reconciledRun.Status.GetCondition(duckv1alpha1.ConditionSucceeded))
 	}
 }
+
+func validateEvents(t *testing.T, r *record.FakeRecorder) {
```

Comment:
Could you add `t.Helper()` in `validateEvents`?
If test errors out in this function it provides better line information

Reply:
Sure.